CMSgov/ato-blueprint
# Styling component library page.

Jira: https://jiraent.cms.gov/browse/ISPGBSS-717

QA:
- Go to Component Library (/controls/components)
- Click on any Component
- The components details page should match the Figma designs (https://www.figma.com/file/9a26GjOwssW76cTr01hfXL/Guidelines-%26-Wireframes?node-id=333%3A2486)
